### PR TITLE
Label configuration hash instead of just a symbol

### PR DESCRIPTION
**Pull request by andresgutgon**
Explained with more details in [this issue](https://github.com/t27duck/active_reporting/issues/15)

### CODE REVIEW

**Comment by t27duck** [comment]
Looks good so far. Thanks for your contribution... There's a couple issues that I'd like to see changed/resolved...

So, this is specifically to add the ability to change the key for a hierarchical dimension? I'm not sure I'm 100% on bored with allowing the use to completely change what the key is, especially if it only applies to hierarchical dimensions. Would a possible compromise be (using the example in your issue) instead of having the key be `gender`, could we always combine the dimension name + the hierarchical level (making it `employee_gender`)? If not, I'd rather have allow the user to override the key for ANY dimension to be whatever they want.

Also, the README should probably be updated to reflect whatever change/option you are adding, if possible.

**Comment by andresgutgon** [comment]
Hi @t27duck thanks for your quick response!
I tried to fix your concerns. Let me know if it's ok now

File: test/fact_models.rb [hunk 1]
```diff
@@ -23,6 +23,7 @@ class GameFactModel < ActiveReporting::FactModel
 end
 
 class SeriesFactModel < ActiveReporting::FactModel
+  dimension_hierarchy %i[name]
```
Comment (t27duck) [inline test/fact_models.rb]: I'd prefer coming up with another test case for this change that doesn't involve adding a hierarchical dimension to this fact model as this is forcing other unrelated tests to change.

Comment (andresgutgon) [reply test/fact_models.rb]: done, platform's hierarchy was not used in any test

**Pull request merged**